PowerPointLabs/PowerPointLabs/ColorsLab/ColorsLabPaneWPF.xaml.cs

```diff
@@ -1252,8 +1292,62 @@ private void SaveDefaultColorPaneThemeColors()
 
         #endregion
 
+        #region Recent Colors
+
+        /// <summary>
+        /// Clear the panel to all white color.
+        /// </summary>
+        private void EmptyRecentColorsPanel()
+        {
+            try
+            {
+                if (this.GetCurrentSlide() != null)
+                {
+                    dataSource.RecentColorOne = Color.White;
```

Comment:
Would you be able to loop through `GetFavoriteColorsPanelAsList()` instead?

Reply:
@leeyh20 I'm not really sure what you mean here... `EmptyRecentColorsPanel()` will set all the recent colors to White, while `GetFavouriteColorsPanelAsList()` returns the favourite colors as a `List<Color>`. They are not referring to the same panel.